river/View.zig

```diff
@@ -36,6 +36,8 @@ const XwaylandView = @import("XwaylandView.zig");
 
 const log = std.log.scoped(.view);
 
+pub var nMainCount: u64 = 1;
```

Comment:
This is the wrong place to store this, IMO.

It'd be better to convert `AttachMode` from `enum` to `union(enum)`, which can then hold the index directly.

```zig
pub const AttachMode = union(enum) {
    top: void,
    bottom: void,
    aside: usize,
};
```

Also the attach mode would then need to be a per-output configuration, which means instead of a single `attach-mode` command there would need to be two commands, `output-attach-mode` and `default-attach-mode`, similar to how layout namespaces work.